### PR TITLE
container name to object, volume to string

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
```SUMMARY: none```

#### Purpose of change
Contributes to #36611 and #36110

#### Describe the solution
Change name to object
Change volume to string and add volume

#### Describe alternatives you've considered
Not changing

#### Testing
image
image
image

### CODE REVIEW

File: data/json/items/containers.json [hunk 1]
```diff
@@ -167,11 +167,11 @@
     "id": "bag_iv",
     "type": "CONTAINER",
     "category": "container",
-    "name": "IV bag",
+    "name": { "str": "IV bag" },
     "description": "A small, sealed plastic bag for liquids used in intravenous therapy.",
     "looks_like": "bag_plastic",
     "weight": "10 g",
-    "volume": 0,
+    "volume": "2 ml",
```
Comment (I-am-Erk) [inline data/json/items/containers.json]:
```suggestion
    "volume": "50 ml",
```

File: data/json/items/containers.json [hunk 2]
```diff
@@ -1135,10 +1135,10 @@
     "id": "condom",
     "type": "CONTAINER",
     "category": "container",
-    "name": "condom",
+    "name": { "str": "condom" },
     "description": "A gentleman's balloon.  A single use life preventer.  A thumbless latex mitten.  This could be used as a makeshift water container, but otherwise it's anyone's guess what it's for.",
     "weight": "5 g",
-    "volume": 0,
+    "volume": "1 ml",
```
Comment (I-am-Erk) [inline data/json/items/containers.json]:
```suggestion
    "volume": "15 ml",
```

File: data/json/items/tool/container.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
     "id": "bottle_folding",
     "type": "CONTAINER",
     "category": "other",
-    "name": "foldable plastic bottle",
+    "name": { "str": "foldable plastic bottle" },
     "description": "A non-rigid plastic bottle for easy storage, holds 500 ml of liquid.",
     "weight": "13 g",
-    "volume": 0,
+    "volume": "3 ml",
```
Comment (I-am-Erk) [inline data/json/items/tool/container.json]:
```suggestion
    "volume": "50 ml",
```

**Comment by snipercup** [comment]
Changes committed thanks I-am-Erk

File: data/json/items/tool/med.json [hunk 4]
```diff
@@ -158,7 +158,7 @@
     "id": "vacutainer",
     "type": "CONTAINER",
     "category": "tools",
-    "name": "blood draw kit",
+    "name": { "str": "blood draw kit" },
     "description": "This is a kit for drawing blood, including a test tube for holding the sample.  Use this tool to draw blood, either from yourself or from a corpse you are standing on.",
     "weight": "13 g",
     "volume": "250 ml",
```
Comment (cainiaowu) [inline data/json/items/tool/med.json]: this seems way to much for normal blood draw kit.

Comment (snipercup) [reply data/json/items/tool/med.json]: The volume?i did not change that. Im just changing the name. It will have to be resoved in another pr.

**Pull request merged**